sys/arm64/arm64/exec_machdep.c

```diff
@@ -431,6 +431,14 @@ fill_capregs(struct thread *td, struct capreg *regs)
 	regs->clr = frame->tf_lr;
 	regs->celr = frame->tf_elr;
 	regs->ddc = frame->tf_ddc;
+	if (td == curthread) {
```

Comment:
Stylistically I would probably do this block first in the function before filling out any of `regs`.

Reply:
Eh, I put it here because it's next to where the PCB gets used, but I take your point, and it's true that style wants a blank line after the } which would make it look a bit odd, so I've moved it.